scripts/ci/update-version-list/main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/thoas/go-funk"
+
+	"github.com/ActiveState/cli/internal/condition"
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/ActiveState/cli/internal/httputil"
+)
+
+// Where the master version file lives on S3.
+const S3PrefixURL = "https://state-tool.s3.amazonaws.com/"
+const S3Bucket = "update/state/"
+const VersionsJson = "versions.json"
+
+// Valid channels to update the master version file with.
+var ValidChannels = []string{"beta", "release", "LTS"}
```

Comment:
You could use the constants here:

https://github.com/ActiveState/cli/blob/a6674251904f1bbbc8ee3fd8798ff8532117dceb/internal/constants/constants.go#L231-L231

Also suggest dropping LTS as it never went into use.